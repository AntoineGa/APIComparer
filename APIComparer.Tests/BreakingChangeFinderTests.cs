using System;
using ApprovalTests;
using APIComparer;
using APIComparer.BreakingChanges;
using NUnit.Framework;

[TestFixture]
internal class BreakingChangeFinderTests
{
    [Test]
    public void ApproveExample()
    {
        var engine = new ComparerEngine();
        var diff = engine.CreateDiff("ExampleV1.dll", "ExampleV2.dll");

        var breakingChanges = BreakingChangeFinder.Find(diff);
        Approvals.Verify(string.Join(Environment.NewLine, breakingChanges));
    }
}