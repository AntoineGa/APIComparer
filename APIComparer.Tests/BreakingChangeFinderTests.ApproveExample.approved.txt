Type removed: Example.MissingNextVersionClass
Type made non public: Example.InternalNextVersionClass